stdlib/test/builtin/test_tuple.mojo

```diff
@@ -56,13 +56,12 @@ def test_tuple_contains():
     var d = (123, True, String("Mojo is awesome"))
 
     assert_true(String("Mojo is awesome") in d)
+    assert_false("Mojo is awesome" in d)
     assert_true(d.__contains__(String("Mojo is awesome")))
 
     assert_false(String("Hello world") in d)
     assert_false(d.__contains__(String("Hello world")))
 
 
 def main():
-    # FIXME(MSTDL-516)
```

Comment:
Don't fully enable this on Mac quite yet. There's a double free we need to chase down in one of these tests with yesterday's nightly.

Reply:
No worries, your fix actually fixed the double free issue we were seeing. @Dan13llljws has a PR internally to re-enable this on mac.  Thanks!